test/integration/index.js

```diff
@@ -734,6 +734,68 @@ describe('bootstrap()', () => {
           contentTypeOpts.should.eql('nosniff');
         });
     });
+
+    it('should not disable caching if noCache is not set', () => {
+      const bs = bootstrap({
+        fields: 'fields',
+        routes: [{
+          views: `${root}/apps/app_1/views`,
+          steps: {
+            '/one': {}
+          }
+        }]
+      });
+
+      return request(bs.server)
+        .get('/one')
+        .expect((res) => {
+          expect(res.headers['cache-control']).to.be.undefined;
+          expect(res.headers.pragma).to.be.undefined;
+        });
+    });
+
+    it('should not disable caching if noCache is false', () => {
+      const bs = bootstrap({
+        fields: 'fields',
+        routes: [{
+          views: `${root}/apps/app_1/views`,
+          steps: {
+            '/one': {}
+          }
+        }],
+        noCache: 'false'
+      });
+
+      return request(bs.server)
+        .get('/one')
+        .expect((res) => {
+          expect(res.headers['cache-control']).to.be.undefined;
+          expect(res.headers.pragma).to.be.undefined;
+        });
+    });
+
+    it('should disable caching if noCache is true', () => {
+      const bs = bootstrap({
+        fields: 'fields',
+        routes: [{
+          views: `${root}/apps/app_1/views`,
+          steps: {
+            '/one': {}
+          }
+        }],
+        noCache: 'true'
```

Comment:
While this is still truthy, would it be more appropriate to have the boolean as type boolean rather than type string?